tests/benchmark/spark/3_GenerateResult.sh

```diff
@@ -10,7 +10,7 @@ latestProp=$directory/latestProp.props
 
 cd $SnappyData/snappy-spark
 echo snappy-spark = $(git rev-parse HEAD)_$(git log -1 --format=%cd) >> $latestProp
-cd ../build-artifacts/scala-2.10/snappy/benchmark/snappy/
```

Comment:
2.11 now

Reply:
I have commented out this command from script as its not required in case of spark. Will remove it from the file.